<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>10</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;MODELO BICAPA&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>891</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_0">
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>256</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>371</width>
        <height>120</height>
       </rect>
      </property>
      <property name="title">
       <string>Datos</string>
      </property>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>27</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Archivo</string>
       </property>
      </widget>
      <widget class="QLabel" name="path_lb">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>341</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgba(0, 0, 0, 140);
background-color: rgb(255, 255, 255);
font: 10pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Path</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_e">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>20</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Explorar</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_g">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>140</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Graficar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_c">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Calcular</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>30</y>
        <width>211</width>
        <height>120</height>
       </rect>
      </property>
      <property name="title">
       <string>Valores iniciales sugeridos</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_13">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>133</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>1000</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>133</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>1000</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>133</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>20</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>83</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>h</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>55</y>
         <width>84</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>rho_2</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>27</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string> rho_1</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>30</y>
        <width>171</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Resultado del modelo</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>h</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>rho_2</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string> rho_1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_rho_1">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgba(0, 0, 0, 140);
background-color: rgb(255, 255, 255);
font: 10pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_rho_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgba(0, 0, 0, 140);
background-color: rgb(255, 255, 255);
font: 10pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_h">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>90</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgba(0, 0, 0, 140);
background-color: rgb(255, 255, 255);
font: 10pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>210</y>
        <width>321</width>
        <height>120</height>
       </rect>
      </property>
      <property name="title">
       <string>Iteraciones y error</string>
      </property>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Gradiente del error</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>62</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Número de evaluaciones FO</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>34</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Número de iteraciones</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_nit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>30</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgba(0, 0, 0, 140);
background-color: rgb(255, 255, 255);
font: 10pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_fev">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>60</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgba(0, 0, 0, 140);
background-color: rgb(255, 255, 255);
font: 10pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_grad">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>90</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgba(0, 0, 0, 140);
background-color: rgb(255, 255, 255);
font: 10pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_1">
     <widget class="QWidget" name="widget_it" native="true">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>70</y>
        <width>441</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget_res" native="true">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>70</y>
        <width>381</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_c_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>480</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Calcular</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>15</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Comparación entre los datos de Resistividad Aparentes</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>40</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculados vs Medidos</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Convergencia de parámetros</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
